media/src/androidMain/kotlin/dev/icerock/moko/media/picker/MediaPickerController.kt

```diff
@@ -49,6 +53,10 @@ actual class MediaPickerController(
     }
 
     actual suspend fun pickImage(source: MediaSource): Bitmap {
+        return pickImage(source, DEFAULT_MAX_IMAGE_WIDTH, DEFAULT_MAX_IMAGE_HEIGHT)
+    }
+
+    actual suspend fun pickImage(source: MediaSource, maxWidth: Int, maxHeight: Int): Bitmap {
```

Comment:
please add comment about broken default arguments in suspend

Reply:
Done.